diagnostics/conf/routes

```diff
@@ -11,7 +11,10 @@ GET        /counts.gif                      controllers.DiagnosticsController.an
 GET        /robots.txt                      controllers.Assets.at(path="/public", file="robots.txt")
 
 POST       /css                             controllers.DiagnosticsController.css
-OPTIONS    /css                             controllers.DiagnosticsController.cssOptions
+OPTIONS    /css                             controllers.DiagnosticsController.postOptions
+
+POST       /csp-report                      controllers.DiagnosticsController.csp
```

Comment:
I'm pretty sure diagnostics runs on beacon.guim.co.uk in prod, but in your fastly PR it'll go to www.theguardian.com, so not sure if this will work